media-fonts/inconsolata-lgc/inconsolata-lgc-1.1.0_p20140917.ebuild

```diff
@@ -7,7 +7,7 @@ inherit font
 EGIT_COMMIT="d086a618248121d61e3f3de64b4301230d1c860c"
 DESCRIPTION="Modified version of Inconsolata Hellenic adding the Cyrillic alphabet"
 HOMEPAGE="https://github.com/DeLaGuardo/Inconsolata-LGC"
-SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/tarball/${EGIT_COMMIT} -> ${P}.tar.gz"
+SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/archive/${EGIT_COMMIT} -> ${P}.tar.gz"
```

Comment:
1: You need to add the `.tar.gz` after `${EGIT_COMMIT`. 
2: It will most likely have a different manifest hash, that's why you need to rename target tarball for now, like `-> ${P}-gh.tar.gz`
3: Due to the updated manifest, you need to commit updated Manifest file as well.